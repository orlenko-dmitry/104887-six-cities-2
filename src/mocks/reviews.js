const reviews = [
  {
    id: 1,
    avatar: `img/avatar-max.jpg`,
    userName: `Max`,
    rating: 4.9,
    message: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
              The building is green and from 18th century.`,
    dateTime: `2019-04-24`,
    dateString: `April 2019`
  },
  {
    id: 2,
    avatar: `img/avatar-angelina.jpg`,
    userName: `Dasha`,
    rating: 4.5,
    message: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
              The building is green and from 18th century.`,
    dateTime: `2019-03-20`,
    dateString: `Mart 2019`
  }
];

export default reviews;
