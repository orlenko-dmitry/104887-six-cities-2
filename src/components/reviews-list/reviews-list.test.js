import React from 'react';
import renderer from 'react-test-renderer';

import ReviewsList from '../reviews-list/reviews-list.jsx';
import reviews from '../../mocks/reviews.js';

it(`ReviewsList renders correctly`, () => {
  const tree = renderer.create(<ReviewsList reviews={reviews} />);

  expect(tree).toMatchSnapshot();
});
