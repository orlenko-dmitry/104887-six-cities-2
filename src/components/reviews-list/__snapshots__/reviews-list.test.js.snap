// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsList renders correctly 1`] = `
<ul
  className="reviews__list"
>
  <ReviewsItem
    key="2"
    review={
      Object {
        "comment": "I stayed here for one night and it was an unpleasant experience.",
        "date": "2019-12-02T13:10:30.195Z",
        "id": 2,
        "rating": 2,
        "user": Object {
          "avatarUrl": "https://htmlacademy-react-2.appspot.com/six-cities/static/avatar/2.jpg",
          "id": 11,
          "isPro": false,
          "name": "Jack",
        },
      }
    }
  />
  <ReviewsItem
    key="1"
    review={
      Object {
        "comment": "The deluxe room was a quite comfortable one with all the adequate facilities. The only thing that made me feel uncomfortable was the rude behavior of an impolite staff at the reception desk.",
        "date": "2019-12-02T13:10:30.195Z",
        "id": 1,
        "rating": 3,
        "user": Object {
          "avatarUrl": "https://htmlacademy-react-2.appspot.com/six-cities/static/avatar/10.jpg",
          "id": 19,
          "isPro": false,
          "name": "Christina",
        },
      }
    }
  />
</ul>
`;
